src/components/Dropdown/Dropdown.scss

```diff
@@ -17,6 +17,18 @@ $border-width: 1px;
     margin-right: var(--spacing-nano);
 }
 
+.moonstone-disabled {
```

Comment:
It seems we can't disable an item with the `TreeViewMenu`.
- The prop `isDisabled` is not defined in the type`TreeViewData`
- Please, can you provide an example in storybook

Reply:
- isDisabled now added
- there's already an example in storybook ( playground / tree)